sound/soc/sof/intel/telemetry.h

```diff
@@ -0,0 +1,45 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2023 Intel Corporation. All rights reserved.
+ *
+ * telemetry data in debug windows
+ */
+
+#ifndef _SOF_INTEL_TELEMETRY_H
+#define _SOF_INTEL_TELEMETRY_H
+
+#define XTENSA_TOOL_CHAIN_ZEPHYR	1
+#define XTENSA_TOOL_CHAIN_XCC		2
+#define XTENSA_SOC_INTEL_ADSP		3
+#define XTENSA_BLOCK_HDR_VER		2
+#define XTENSA_CORE_DUMP_SEPARATOR	0x0DEC0DEB
+#define XTENSA_CORE_AR_REGS_COUNT	16
+
+struct xtensa_arch_block {
```

Comment:
I've invited a few extra persons to take a look at this PR. I think looks good, but I like to get some double-checks whether we can build on these definitions. I believe these are used by the gdb clients, so the interface is stable, but let us check.

Reply:
We are good with this. If there are changes, the version field will change.